docs_guidelines/how_to/write_mathematical_expressions.rst

```diff
@@ -0,0 +1,42 @@
+==============================
+Write mathematical expressions
+==============================
+
+This guide shows how to add :math:`\LaTeX` expressions to documentation.
+
+Pre-requisites
+==============
+
+This guide assumes your Qiskit project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+`sphinx-quickstart <https://www.sphinx-doc.org/en/master/man/sphinx-quickstart.html>`_.
+
```

Comment:
similar to `create_lists.rst` I think pre-requisites are not needed if the guide is a pure rst guide.